core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -133,6 +137,19 @@ var PreferencesPage = function() {
   this.expectUserBioToBe = function(bio) {
     expect(userBioElement.getAttribute('value')).toMatch(bio);
   };
+
+  this.selectCreatorDashboard = function() {
+    createrDashboardRadio.click();
+  };
+
+  this.selectLearnerDashboard = function() {
+    learnerDashboardRadio.click();
+  };
+
+  this.goToHomePage = function() {
+    browser.get('/');
+    return waitFor.pageToFullyLoad();
+  };
```

Comment:
Shouldn't this be I `general.js` instead, is seems as a quite general method.

Reply:
Done